types/datum_test.go

```diff
@@ -194,6 +198,57 @@ func (ts *testTypeConvertSuite) TestToFloat32(c *C) {
 	c.Assert(converted.GetFloat64(), Equals, datum.GetFloat64())
 }
 
+func (ts *testTypeConvertSuite) TestToFloat64(c *C) {
+	var datum = NewDatum(float32(3.00))
```

Comment:
Can we extract to a function? There are too many redundant code. just like `TestStrToDuration`.

Reply:
Any advice for me?
There are different types, It is a little hard for me to use table-driven tests...

Reply:
Done.